src/layerInfos/BarrierfreierLayerInfo/BarrierfreierLayerInfo.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+
+import './BarrierfreierLayerInfo.scss';
+
+const propTypes = {
+  t: PropTypes.func.isRequired,
+  language: PropTypes.string.isRequired,
+};
+
+const defaultProps = {};
+
+const desc = {
+  de: (
+    <div>
+      Barrierefreier Bahnhof an dem beim SBB Call Center Handicap unter der
+      Nummer <a href="tel:0800 007 102">0800 007 102</a> eine Hilfestellung
+      bestellt werden kann.
+    </div>
+  ),
```

Comment:
I recommend to create only one LayerInfo component for all three handicap layers. The following text is always the same.

> beim SBB Call Center Handicap unter der Nummer ... eine Hilfestellung berstellt werden kann.

You could solve it similar to the ConstructionLayerInfo, see [layer configuration](https://github.com/geops/trafimage-maps/blob/master/src/config/layers.js#L529) (including the lines below) and [ConstructionLayerInfo.js](https://github.com/geops/trafimage-maps/blob/master/src/layerInfos/ConstructionLayerInfo/ConstructionLayerInfo.js)

Reply:
unique LayerInfo file used